docs/Makefile

```diff
@@ -157,6 +157,12 @@ gettext:
 	@echo
 	@echo "Build finished. The message catalogs are in $(BUILDDIR)/locale."
 
+intl-%: gettext
+	( \
+	MODIFIED=$$(sphinx-intl update -p $(BUILDDIR)/locale -l $(subst intl-,,$@) | tee /dev/stderr | grep -e 'Update:' -e 'Create:' | cut -f 2 -d' '); \
+	powrap $$MODIFIED; \
```

Comment:
That would work most of the time:

On my side I'm using the docker container (builded from `docs/Dockerfile`). 
And to be able to use `--modified` I would need to mount the whole `parsec-cloud` folder instead of only `docs`.

Reply:
I see, we can leave it that way here. 

However, we will probably need a `powrap --check` in the ci (or pre-commit :thinking:) because someone may fix or add a translation directly in the `.po` file and forget to apply powrap. Maybe we can open another issue/PR for that.

Reply:
I've created #5648